src/olympia/files/templates/files/viewer.html

```diff
@@ -18,6 +18,18 @@
      {% if automated_signing %}data-automated-signing="true"{% endif %}
      data-validation-failed="{{ _("Validation failed:") }}"></div>
 
+<div class="modal highlighter-output-broken js-hidde">
+  <h2>{{ _('File content not supported') }}</h2>
```

Comment:
I'm sorry for the extra round, but I think `File content display not supported` would be a bit more descriptive. Just a suggestion, though.